EpikCord/ext/tasks.py

```diff
@@ -0,0 +1,82 @@
+import datetime
+from typing import Callable, Optional
+import asyncio
+from ..exceptions import TaskFailedError
+
+_EPOCH = datetime.datetime(1970, 1, 1)
+__all__ = (
+    "TimeParser",
+    "Tasks",
+)
+
+
+
+
+@staticmethod
```

Comment:
Isn't this only required for classes where the function doesn't need `self`

Reply:
Oops, 4got to remove it after deleting the parent class